<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPillDialog</class>
 <widget class="QDialog" name="addPillDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Название:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pillNameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Правила приема</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="pillBeforeRadioButton">
              <property name="text">
               <string>До еды</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="pillAfterRadioButton">
              <property name="text">
               <string>После еды</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="pillBreakfastCheckBox">
              <property name="text">
               <string>Завтрак</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="pillLunchCheckBox">
              <property name="text">
               <string>Обед</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="pillDinnerCheckBox">
              <property name="text">
               <string>Ужин</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Доза:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="amountLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Срок приёма:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pillLifeTimeSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>дней</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="OKPushButton">
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>addPillDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OKPushButton</sender>
   <signal>clicked()</signal>
   <receiver>addPillDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
